lib/src/participant/local_participant.dart

```diff
@@ -230,3 +232,46 @@ class LocalParticipant extends Participant {
     super.updateFromInfo(info);
   }
 }
+
+extension LocalParticipantTrackSourceExt on LocalParticipant {
+  Future<void> setCamercaEnabled(bool enabled) async {
```

Comment:
nit: typo in `setCamercaEnabled` (should be `setCameraEnabled`)

Reply:
Omg yesss thanks onichan!!! 🙏🏻😁